tests/pkg/tests/observability_alert_test.go

```diff
@@ -28,6 +28,10 @@ import (
 	"github.com/stolostron/multicluster-observability-operator/tests/pkg/utils"
 )
 
+const (
+	TrueStr = "true"
+)
```

Comment:
Do we really need an exported const here for this? 😅 

I see it being used in many places, but I've never seen this pattern before.

Reply:
Ah, I remember something screamed at me in my IDE to do this. 😆 
But probably doesn't need to be exported.